lib/src/ioctl.rs

```diff
@@ -359,9 +359,10 @@ pub struct V4l2Buffer {
     planes: V4l2BufferPlanes,
 }
 
-/// V4l2Buffer is safe to send across threads. `v4l2_buffer` is !Send because it
-/// contains a pointer, but we are making sure to use it safely here.
+/// V4l2Buffer is safe to send across threads. `v4l2_buffer` is !Send & !Sync
```

Comment:
Shouldn't this read `Send & Sync`? Looks like I did a typo in the initial comment...

Reply:
Rust pointers are neither Send nor Sync, the comment is/was correct I'd say?

Reply:
You're absolutely right, I've misread my own comment. :)